src/tests/account/ethereum/test_eth_account.cairo

```diff
@@ -46,32 +45,27 @@ fn COMPONENT_STATE() -> ComponentState {
     EthAccountComponent::component_state_for_testing()
 }
 
-fn setup() -> ComponentState {
+fn setup(key_pair: Secp256k1KeyPair) -> ComponentState {
```

Comment:
I think having  a key pair passed explicitly is better, since in many test cases the key pair is also used for result validation. If KEY_PAIR constant was used implicitly in the `setup` function, then 
- it would introduce an assumption that KEY_PAIR constant is considered to be the default one
- we would have to build/compute key pair twice, in the `setup` fn and in the test itself